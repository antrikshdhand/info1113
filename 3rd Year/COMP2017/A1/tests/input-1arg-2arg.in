place H2
place J3
who
history
place N2
who ButTheresASecondArgument!
who ^^^
view jk
resign no
