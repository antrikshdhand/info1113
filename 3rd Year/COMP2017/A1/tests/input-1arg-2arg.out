B
H2J3
Invalid!
Invalid!
Invalid!
Invalid!
